README.md

```diff
@@ -55,6 +55,28 @@ printAfter(function () {
 });
 ```
 
+### runAsync.cb
+
+`runAsync.cb` can be used for functions that have a constant number of parameters. It supports all the function types that `runAsync` does and additionally a traditional **callback as the last argument** signature:
+
+```js
+var runAsync = require('run-async');
+
+runAsync.cb(function(a, b, cb) {
+  cb(null, a + b);
+}, function(err, result) {
+  console.log(result)
+})(1, 2)
+
+// this won't work though, because the function has an arbitrary number of arguments
```

Comment:
Lets not add broken example.

Lets just state that it'll only work if a function takes a fixed number of parameters. (Remove that part from the intro and move it later in the body)